src/pages/spawn/spawnVolume/spawnVolumeTable/SpawnVolumeCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { CardContainer, CardField, DoesNotExpire } from "components/Spawn";
+import { MyVolume } from "types/spawn";
+import { getDateCopy } from "utils/string";
+
+interface Props {
+  volume: MyVolume;
+}
+
+export const SpawnVolumeCard: React.FC<Props> = ({ volume }) => (
+  <CardContainer
+    data-cy={`spawn-volume-card-${volume.displayName || volume.id}`}
+  >
+    {spawnVolumeCardFields.map(({ label, Comp }) => (
+      <CardField key={label} field={label} value={<Comp volume={volume} />} />
+    ))}
+  </CardContainer>
+);
+
+interface CardItem {
+  label: string;
+  Comp: React.FC<Props>;
+}
+
+const spawnVolumeCardFields: CardItem[] = [
+  {
+    label: "Created at",
+    Comp: ({ volume }) => <>{getDateCopy(volume.creationTime)}</>,
+  },
+  {
+    label: "Expires at",
+    Comp: ({ volume }) => (
+      <>
+        {volume.noExpiration || !volume.expiration ? (
+          <DoesNotExpire />
+        ) : (
+          getDateCopy(volume.expiration)
+        )}
+      </>
+    ),
+  },
+  { label: "Type", Comp: ({ volume }) => <>{volume.type}</> },
+  { label: "Size", Comp: ({ volume }) => <>{volume.size} GB</> },
+  {
+    label: "Availability Zone",
+    Comp: ({ volume }) => <>{volume.availabilityZone}</>,
+  },
+  {
+    label: "Is Home MyVolume",
+    Comp: ({ volume }) => <>{volume.homeVolume ? "True" : "False"}</>,
```

Comment:
This volume which isn't mounted to anything is returning true for this field. Not sure if thats intended or not.
![image](https://user-images.githubusercontent.com/4605522/94305398-15ae7a00-ff3f-11ea-9aae-e9820c10ca4d.png)

Reply:
Which environment is the app server you are pointing to? I was unable to reproduce this by 

1. creating a volume from here: https://evergreen.mongodb.com/spawn#?resourcetype=volumes
2. pulling this branch and pointing gql requests to the production app server
3. opening the corresponding volume card at http://localhost:3000/spawn/volume

Reply:
I think i see whats happening. In order to reproduce. 
Spawn a host with a volume. 
Then terminate the host the volume remains.